src/core_plugins/kibana/public/management/sections/indices/create_index_pattern_wizard/step_time_field/step_time_field.html

```diff
@@ -0,0 +1,136 @@
+<form
+  role="form"
+  ng-submit="stepTimeField.createIndexPattern()"
+  class="kuiPanel"
+>
+  <div class="kuiPanelHeader">
+    <div class="kuiPanelHeaderSection">
+      <h2 class="kuiPanelHeader__title">
+        Step 2 of 2: Optional settings
+      </h2>
+    </div>
+  </div>
+
+  <div class="kuiPanelBody">
+    <p class="kuiText kuiSubduedText kuiVerticalRhythm">
+      You've defined <strong>{{stepTimeField.indexPatternName}}</strong> as your index pattern. Now you can specify some optional settings before we create it.
+      <small>
+        <a
+          class="kuiLink"
+          ng-click="stepTimeField.toggleAdvancedOptions()"
+        >
```

Comment:
This isn't keyboard accessible (can't be tabbed to) because it doesn't have an `href` on it. We just need to add `kbn-accessible-click` as an attribute to solve this problem.

I'd suggest removing the `<small>` element and placing this link and the "index pattern name" field beneath the time field filter name field, but I'd also like to hear @snide's thoughts.